docker/phpfpm/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM php:7.2-fpm
+
+RUN usermod -u 1000 www-data
+RUN usermod -G staff www-data
+
+RUN apt-get update && apt-get install -y libpng-dev libjpeg-dev libpq-dev zip unzip sudo wget sqlite3 libsqlite3-dev && rm -rf /var/lib/apt/lists/*
+
+RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr
+RUN docker-php-ext-configure pdo_mysql --with-pdo-mysql=mysqlnd
+RUN docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql
+RUN docker-php-ext-configure mysqli --with-mysqli=mysqlnd
+
+RUN yes | pecl install igbinary redis
```

Comment:
This runs on error as `libzstd` is required for this Redis version (explicit versioning to the rescue):
`
checking for libzstd files in default path... not found
configure: error: Please reinstall the libzstd distribution
ERROR: /tmp/pear/temp/redis/configure --with-php-config=/usr/local/bin/php-config --enable-redis-igbinary=y --enable-redis-lzf=y --enable-redis-zstd=y' failed
ERROR: Service 'tao_phpfpm' failed to build: The command '/bin/sh -c yes | pecl install igbinary redis' returned a non-zero code: 1
`

Reply:
@jbout should be fixed into develop later